nevergrad/benchmark/experiments.py

```diff
@@ -86,6 +87,22 @@ def keras_tuning(
     # First, a few functions with constraints.
     optims: tp.List[str] = ["PSO", "OnePlusOne"] + get_optimizers("basics", seed=next(seedg))  # type: ignore
     datasets = ["kerasBoston", "diabetes", "auto-mpg", "red-wine", "white-wine"]
+    optims += [
+        "Shiwa",
+        "NGO",
+        "NGOpt4",
+        "NGOpt8",
+        "NGOpt10",
+        "NGOpt12",
+        "NGOpt13",
+        "NGOpt14",
+        "NGOpt15",
+        "NGOpt16",
+        "NGOpt21",
+        "NGOpt36",
+        "NGOpt38",
+        "NGOpt",
```

Comment:
please use an optimizer group instead, that will avoid duplication

Reply:
Done.

Reply:
only in 1 place out of 2 it seems